recipe/patches/0009-Use-system-mkldnn.patch

```diff
@@ -0,0 +1,51 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 98593c2d..94a9d63d 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
```

Comment:
For quickly applying the patches as they exist on the feedstock (i.e. if it's not you who did the last change to them), it's nicer to generate them with `git format-patch <tag> --no-signature`.

That has several advantages:
* commit message can be used to provide context (this patch is pretty self-explanatory, but in general...)
* authorship is easily visible (who to ask in case there are questions about intent, e.g. when rebasing to new versions)
* easily apply them to a source-checkout in one go using e.g.
  `find ../path/to/feedstock/recipe/patches/ | xargs git am`
* having a local branch of patches makes it much easier to rebase the changes to a new version
* etc.

So while raw diff's are technically possible, the output of `git format-patch` is preferred.